Assets/Src/SceneTransitionCommands/ISceneTransitionCommand.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Src.SceneTransitionCommands
+{
+    public interface ISceneTransitionCommand
```

Comment:
Maybe it would be better to name it like AuthCompletionHandler or AuthEndHandler or something like that.